lib/src/layer/tile_layer/tile_layer.dart

```diff
@@ -706,6 +714,11 @@ class _TileLayerState extends State<TileLayer> with TickerProviderStateMixin {
 
   bool _outsideZoomLimits(num zoom) =>
       zoom < widget.minZoom || zoom > widget.maxZoom;
+
+  void _resetStreamHandler(void _) {
+    _tileImageManager.removeAll(widget.evictErrorTileStrategy);
+    if (mounted) _loadAndPruneInVisibleBounds(MapCamera.of(context));
```

Comment:
Does this handle the tile buffer around the visible tiles, too?

Reply:
This doesn't change the reset logic, it only ensures the handler is fired. Another PR is needed to look into this and fix #1813.